src/content/governance/index.md

```diff
@@ -0,0 +1,159 @@
+
+## Informal Process {#informal-process}
+
+### Understanding Prior Work {#prior-work}
+
+EIP Champions should familiarise themselves with prior work and proposals before creating an EIP which can be seriously considered for deployment on the Ethereum Mainnet. This way, the EIP hopefully brings something new which hasn't been rejected before. The three main places to research this are the [EIP repository](https://github.com/ethereum/eips), [Ethereum Magicians](https://www.ethereum-magicians.org/) and [ethresear.ch](https://www.ethresear.ch/).
+
+### Working Groups {#working-groups}
+
+An initial EIP is unlikely to be implemented on the Ethereum Mainnet without edits or changes. Generally, EIP Champions will work with a subset of Protocol Developers to specify, implement, test, iterate, and finalize their proposal. Historically, these working groups have required several months (and sometimes years!) of work. Similarly, EIP Champions for such changes should involve relevant Application/Tooling Developers early in their efforts to gather end-user feedback and mitigate any deployment risks.
+
+### Community Consensus {#community-consensus}
+
+Unfortunately, there is no clear playbook on how to handle contentious proposals. Since they have no way to force people to adopt network upgrades, Protocol Developers will generally avoid including EIPs that are contentious amongst various stakeholder groups.
+
+As the champion of a contentious EIP, you should try and build consensus, address objections and document community support for your proposal. Given the size and diversity of the Ethereum community, there isn't a single metric (e.g. a coin vote) that can be used, and EIP Champions are expected to adapt to the circumstances of their proposal.
+
+Beyond the security of the Ethereum network, significant weight is usually placed on what Application/Tooling Developers and Application Users value, given that their using and developing on Ethereum is what makes the ecosystem attractive for other stakeholders.
+
+<Divider />
+
+## Handling disagreements {#disagreements}
+
+Having many stakeholders with different motivations and beliefs means that disagreements are not uncommon.
+
+Generally, disagreements are handled with long-form discussion to understand the root of the problem. Typically, one group concedes, or a happy medium is achieved. If one group feels strongly enough, forcing through a particular change could result in a split chain. A split chain is when some stakeholders protest implementing a protocol change resulting in different, incompatible versions of the protocol operating, from which two distinct blockchains emerge.
+
+### The DAO fork {#dao-fork}
+
+Forks are when major technical upgrades or changes need to be made to the network and change the "rules" of the protocol. [Ethereum clients](/developers/docs/nodes-and-clients/) must update their software to implement the new fork rules.
```

Comment:
i'd lay out what forks actually refer to. Network upgrades are defined above in step #4 under **the formal process**

"Sometimes, network upgrades are contentious: these can lead to what are known as "forks:" a point where two sets of nodes start running client software with incompatible protocol rules. Once this happens, the chain splits into two separate pieces, with their own unique histories. Hard Forks are not very common occurrence, but are impactful when they do happen."